extensions/account-registration.md

```diff
@@ -218,6 +223,18 @@ The client can then proceed with the connection:
     S: 001 tester :Welcome to the IRC network
     ...
 
+## With no email or verification
+
+    C: CAP LS 302
+    S: CAP * LS :draft/account-registration=before-connect
+    C: CAP REQ :draft/account-registration
+    C: CAP END
+    ...
+    C: REGISTER * * hunter2
+    S: REGISTER SUCCESS tester :Account successfully registered
+
+Note that email addresses may be required by the server, but not verified.
```

Comment:
What does this mean? As in the server sends email-required but no VERIFICATION_REQUIRED?

Reply:
Yes. Any suggestion to phrase it better?

Reply:
I'd just leave it out, less confusing.

Reply:
Better to just split the examples if you think people might be confused.

Reply:
ok, done